plugins/com.google.cloud.tools.eclipse.login/src/com/google/cloud/tools/eclipse/login/ui/AccountsPanel.java

```diff
@@ -83,9 +77,18 @@ protected Control createDialogArea(Composite parent) {
   @VisibleForTesting
   void createAccountsPane(Composite container) {
     for (Account account : loginService.getAccounts()) {
-      Label label = new Label(container, SWT.NONE);
-      label.setText(account.getEmail());
-      accountLabels.add(label);
+      Label name = new Label(container, SWT.LEAD);
+      if (account.getName() != null) {
+        name.setText(account.getName());
+      }
+      name.setData("org.eclipse.e4.ui.css.CssClassName", "accountName");
```

Comment:
Would it be worth extracting this to a package-protected constant and referencing it from your tests?